Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/Target/Native/TestImpactNativeTestTarget.cpp

```diff
@@ -10,6 +10,12 @@
 
 namespace TestImpact
 {
+    namespace SupportedTestFrameworks
+    {
+        //! The CTest label for the GTest framework.
+        inline constexpr auto GTest = "FRAMEWORK_googletest";
```

Comment:
This type should be set to a `const char*` to be explicit about the type.
Also would deduce this type to a `const char[X]`
```suggestion
        inline constexpr const char* GTest = "FRAMEWORK_googletest";
```